src/pages/confirmation/confirmation.styles.js

```diff
@@ -5,8 +5,8 @@ export const useStyles = makeStyles((theme) => ({
   confirmation: {
     background: `url(${BG}) no-repeat `,
     backgroundSize: 'cover',
-    height: '700px',
-    position: 'relative'
+    backgroundPosition: 'center',
+    minHeight: '100vh'
```

Comment:
a height 100vh is not a good idea, maybe there will be a scroll that we don't want to see on the page

Reply:
fixed